wicket/src/cli/rack_setup/config_template.toml

```diff
@@ -38,6 +38,16 @@ internal_services_ip_pool_ranges = []
 # Confirm this list contains all expected sleds before continuing!
 bootstrap_sleds = []
 
+# Allowlist of source IPs that can make requests to user-facing services.
+[allowed_source_ips]
+# Any external IPs to make requests. This is the default.
+allow = "any"
```

Comment:
There will need to be some small changes to wicketd to accommodate this (add it to `UserSpecifiedRackNetworkConfig`, then pass it through from that to the `RackNetworkConfigV1` we create to send to sled-agent). I actually am not sure wicketd will compile without those change? It shouldn't!

Reply:
This wicketd work seems likely to conflict with #5685, although maybe not terribly so? Just wanted to call it out as an incoming hazard.

Reply:
I was unclear what amount of wicket work would be required. It does seem to compile just fine, though, or at least I am able to `cargo c` all of Omicron without issue! I will dig in more today.

Reply:
I'd also vote `replace` for urgency here. We're using it in some pretty questionable spots already, and we probably have some work to do to unravel that in the future; I don't think this is any worse than any of that.

Reply:
I vote strongly for replace, I think the

Reply:
I vote strongly for replace. I think trying to write conversions just doesn't add anything here.

Reply:
replace wins by a mile